### DISCUSSION TITLE
Wrong inverted colors inside Sidepanel.

### DISCUSSION THREAD

**Discussion by rendomnet** [microsoft/MicrosoftEdge-Extensions#204] [Discussions]
For some reason, my extension appears inverted within the Edge side panel (only the bright colors, not all colors). It seems as though some kind of 'Dark mode' is being applied to my already dark extension. When I use Inspect, the correct colors are displayed in developer console but incorrect color in Sidepanel
. 
This issue is exclusive to the side panel; in a new tab, my colors appear correctly.

This problem doesn't occur in Chrome or other browsers.

**Comment by ManikanthMSFT** [comment]
Hi @rendomnet, thank you for reporting this issue. We are currently looking into it and will update you as soon as we have more information.

In the meantime, could you please provide us with screenshots of the reporting error to assist us in our investigation?

Thank you for your cooperation.

**Comment by rendomnet** [comment]
<img width="387" alt="Screenshot 2024-10-04 at 9 56 56 PM" src="https://github.com/user-attachments/assets/d63cdfce-55a9-421f-ab04-a2dbf3ca76bb">
@ManikanthMSFT 
Here. Toggle indicator should be yellow but for some reason it is dark brown. 
But in developer console it is still yellow.

**Reply by ManikanthMSFT** [reply]
@rendomnet, could you please help us with your extension details like name of your extension, seller ID for us to assist you further.

**Comment by iNick** [comment]
We have a similar issue with our extension where the background color is black in the side panel regardless of the color set in the CSS.

<img width="1081" alt="Skärmavbild 2024-10-09 kl  12 06 58" src="https://github.com/user-attachments/assets/3a1cbf42-a3b3-4a6e-81f9-494f8d62b0cc">

For us this only happens in Edge on Mac. Current version 129.0.2792.65. It appears to be a fairly recent regression, as Edge used to display the correct white background color. But I can't say exactly when this changed. This is not a problem on Edge for Windows. The colors are fine in Chrome with the same extension.

Changing the dark mode settings of the browser has no impact.

**Reply by rishikramena-ms** [reply]
Hi @iNick, can you please share a sample extension which reproduces this issue? I tried a sample extension with a default side panel on the specified version of Edge but could not reproduce it.

**Reply by iNick** [reply]
@rishikramena-ms Here's an example: https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-open

In Edge version 129.0.2792.65:
<img width="1447" alt="Skärmavbild 2024-10-09 kl  14 18 49" src="https://github.com/user-attachments/assets/c2f42f49-8338-402e-aca5-04966cb22001">

In Chrome version 129.0.6668.59:
<img width="1474" alt="Skärmavbild 2024-10-09 kl  14 19 57" src="https://github.com/user-attachments/assets/5aec03ff-3466-456c-9cb4-0fe50f9d73cb">

**Reply by rishikramena-ms** [reply]
![Video](https://github.com/user-attachments/assets/2fa400f5-ea58-433b-85f0-bce4e20548e9)
Can you please review this recording and share if we are missing something which might help reproducing this issue?

**Reply by iNick** [reply]
When I follow the exact same steps as in your recording, the colors of the Global side panel are inverted. Black background, white text.

The only differences in the setup that I can see:
- macOS version
- Arm vs. x86
- Locale

<img width="582" alt="Skärmavbild 2024-10-10 kl  11 02 26" src="https://github.com/user-attachments/assets/2fb9475e-534e-4cdd-9642-2f087cb2a1ef">

**Reply by rishikramena-ms** [reply]
Thanks for this info @iNick. We were able to reproduce this issue and found out that this is only an issue with macOS with ARM chips(M1, M2). We'll investigate what's causing this and get back with an update soon.

**Comment by rajatrocks** [comment]
I'm also experiencing this issue and am on a Mac with an M2 chip. Short video here: https://youtu.be/Y2w_gWwrmBE

**Comment by marloncodum** [comment]
We have the same issue. Problem doesn't seem to be resolved, yet. (Can confirm Macs with M1 and M2 chips.)

**Comment by Arktomson** [comment]
I have also encountered this problem. I am also a Mac system and my computer chip is M3. I hope you can fix it as soon as possible

**Comment by rajatrocks** [comment]
It has been almost 5 months and this still hasn't been resolved? This makes it hard to take Edge seriously.

**Reply by ManikanthMSFT** [reply]
Hi @rendomnet, I sincerely apologize for the delay in resolving your query regarding the Sidepanel issue. I understand your frustration and appreciate your patience during this time.

Please be assured that our team is actively working on this issue. We have created an Incident and are collaborating with the relevant teams to address the problem as quickly as possible. We understand the importance of this matter and are committed to providing a resolution soon.

Thank you for your continued patience and understanding. If you have any further questions or need additional assistance, please do not hesitate to reach out.

**Reply by arrow79** [reply]
Another month has passed, and the problem still exists.

**Reply by ManikanthMSFT** [reply]
Hi @arrow79, we would like to assure you that this issue is a top priority for our team. We are actively working on it and will provide you with updates as soon as we have more information.

**Comment by CrazyBoyM** [comment]
Fuck you for what time you will slove this problem? If you can not slove, pls close the edge browser project!

**Comment by ManikanthMSFT** [comment]
Hi team, we appreciate your patience. We got an update from our team stating that the Bug is fixed in the canary. Once we get more information we will write back to you.
